src/components/radioButton/RadioButton.js

```diff
@@ -58,7 +112,14 @@ class RadioButton extends BaseComponent {
     const {value, disabled} = this.props;
     if (!disabled) {
       _.invoke(context, 'onValueChange', value);
-      _.invoke(this.props, 'onPress', this.isSelected(this.props, context));
+      const selected = this.isSelected(this.props, context);
```

Comment:
Why you're not using `this.selected` here instead of calling `this.isSelected()` again?
Did you check if the value sent to `onPress` here is now the correct one?